examples/xlio_ultra_api_ping_pong.c

```diff
@@ -350,9 +233,7 @@ static int init_xlio_ultra_api(void)
     int rc;
 
     /*
-     * Step 1: Obtain XLIO API function pointers
-     * ==========================================
-     * Use xlio_get_api() from <mellanox/xlio_extra.h> to retrieve the function pointer structure.
+     * Step 1: Obtain XLIO API function pointers.
```

Comment:
Step 1 ends with period, Steps 2 and 3 don't
Step 1 has explanation, Steps 2 and 3 don't.
suggestion:
`    /*
     * Step 1: Obtain XLIO API function pointers
     * Uses xlio_get_api() which works when libxlio is preloaded with LD_PRELOAD.
     */
    
    /*
     * Step 2: Validate API compatibility
     * Check that the loaded library supports XLIO Ultra API.
     */
    
    /*
     * Step 3: Initialize XLIO
     * Set up initialization attributes and call xlio_init_ex().
     */`